components/variables.tf

```diff
@@ -4,20 +4,12 @@ variable "oauth_token" {
   sensitive   = true
 }
 
-variable "branches" {
-  description = "List of branches to apply protection rules"
-  type        = list(string)
-  default = [
-    "master",
-    "main"
-  ]
-}
-
-# variable "excluded_repositories" {
```

Comment:
commented variables - why not just remve?

Reply:
Removed now